suite-native/navigation/src/components/GoBackIcon.tsx

```diff
@@ -5,10 +5,11 @@ import { IconButton } from '@suite-native/atoms';
 import { CloseActionType } from '../navigators';
 
 type GoBackIconProps = {
+    onPress?: () => void;
```

Comment:
I would advise against this if you're sending onPress which doesn't go back, then it shouldn't be in GoBack component :D

Reply:
You are right. The GoBackIcon changes reverted: 3fd9ef1399

Reply:
The navigation will be implemented in the useEffect, that will react to change of the send form error state.

Issue created here: https://github.com/trezor/trezor-suite/issues/13125